### DISCUSSION TITLE
Unable to Load Images from folders

### DISCUSSION THREAD

**Discussion by ColbysCode** [keirf/flashfloppy#828] [General]
I have been trying to load images from a folder I named DOS on the root directory. But it doesn't seem to work and get the code E34. If I place the image files in the root folder, it works fine. at first, I thought it was an issue from me formatting the USB with Linux. so I tried formatting it with Windows 98 and I still have the same issue. 

My FF.CFG file have the 'Nav-mode = native' enabled. I tried searching everywhere, can't find any information on this. Not sure if I'm doing something wrong. Does anybody here have any ideas?

**Comment by keirf** [comment]
Do you have oled display or three digit display?

**Comment by ColbysCode** [comment]
3 digit display.

**Comment by keirf** [comment]
Folder navigation is not supported for three-digit display. See here (but perhaps the description could be clearer!): https://github.com/keirf/FlashFloppy/wiki/Initial-Setup#native-mode

**Comment by ColbysCode** [comment]
that makes sense, thanks for letting me know.
